sonar-scm-git-plugin/src/main/java/org/sonarsource/scm/git/JGitBlameCommand.java

```diff
@@ -75,12 +74,8 @@ public void blame(BlameInput input, BlameOutput output) {
 
   private boolean cloneIsInvalid(File gitBaseDir) {
     if (Files.isRegularFile(gitBaseDir.toPath().resolve(".git/objects/info/alternates"))) {
-      LOG.warn("This repository references another local repository which is not supported. "
+      LOG.warn("This repository references another local repository which is not well supported. SCM information might be missing for some files. "
```

Comment:
Isn't warn a bit too strong? Maybe info is enough?

Reply:
I will change it to info